Dockerfile

```
@@ -14,33 +22,20 @@ RUN --mount=type=secret,id=SENTRY_ORG,env=SENTRY_ORG \
-
-WORKDIR /app
-
-# Cache dependency compilation by building a stub first.
-COPY Cargo.toml Cargo.lock /app/
-RUN mkdir src && echo 'fn main() {}' > src/main.rs && \
-    cargo build --release && \
-    rm -rf src target/release/cbzViewer target/release/deps/cbzViewer*
-
-COPY src /app/src
-RUN cargo build --release
+FROM dunglas/frankenphp
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)